sdk/keyvault/azure-security-keyvault-keys/README.md

```diff
@@ -14,7 +14,7 @@ Maven dependency for the Azure Key Vault Key client library. Add it to your proj
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-keys</artifactId>
-    <version>4.3.0-beta.5</version>
+    <version>4.3.0-beta.6</version>
```

Comment:
Is this version released?

Reply:
Please refer to [this comment](https://github.com/Azure/azure-sdk-for-java/pull/20521#discussion_r610940121).